src/main/java/edu/wpi/first/pathui/PathCell.java

```diff
@@ -46,6 +47,32 @@ public PathCell(boolean validDropTarget, BiFunction<String, String, Boolean> val
     this.setConverter(new DefaultStringConverter());
   }
 
+  @Override
+  public void updateItem(String s, boolean b) {
+    super.updateItem(s, b);
+    updateColor(this.getIndex());
+  }
+
+  @Override
+  public void updateIndex(int i) {
+    super.updateIndex(i);
+    updateColor(i);
+  }
+
+  private void updateColor(int index) {
+    // Determine whether this is a child of an auton (child of a child)
+    if (this.getTreeItem() != null && FxUtils.isSubChild(this.getTreeView(), this.getTreeItem())) {
+      Color color = FxUtils.getColorForSubChild(index);
+      int r = (int) (color.getRed() * 255);
+      int g = (int) (color.getGreen() * 255);
+      int b = (int) (color.getBlue() * 255);
+      String rgbStr = r + ", " + g + ", " + b;
+      this.setStyle("-fx-background-color: rgb(" + rgbStr + ");");
```

Comment:
I wouldn't mix CSS and code like this. You could either set a CSS pseudoclass eg `path-cell:first`, `path-cell:second`, or call `setBackground(new Background(new BackgroundFill(color, null, null))`